coops-ui/tests/unit/AGMDate.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('AGMDate', () => {
   beforeEach(async () => {
     // init store
     store.state.entityIncNo = 'CP0001191'
-    store.state.currentDate = '2019/07/15'
```

Comment:
FYI - this was a left-over from old code. The current code uses dashes so I updated the tests to match.